hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                                    customAlert("<i class='glyphicon glyphicon-flag custom-alert-icon'></i><strong>Collection Status:</strong> sufficient to publish or make public", 3000);
+                                    $("#btn-public").prop("disabled", false);
+                                    $("#btn-discoverable").prop("disabled", false);
+                                }
+                            }
+                        }
+                        $('body > .container').append($alert_success);
+                        $('#error-alert').each(function(){
+                            this.remove();
+                        });
+                        $(".alert-success").fadeTo(2000, 500).fadeOut(1000, function(){
+                            $(document).trigger("submit-success");
+                            $(".alert-success").alert('close');
+                        });
+                    }
+                    else{
+                        $('body > .container').append($alert_error);
+                        $(".alert-danger").fadeTo(3000, 500).fadeOut(1000, function(){
+                            $(document).trigger("submit-error");
+                            $(".alert-danger").alert('close');
+                        });
+                    }
+                },
+                error: function(XMLHttpRequest, textStatus, errorThrown)
+                {
+                    $('#' + form_id).before($alert_error);
+                    $(".alert-error").fadeTo(2000, 500).slideUp(1000, function(){
+                        $(".alert-error").alert('close');
+                    });
+                }
+            });
+            //don't submit the form
+            return false;
+        }
+
+
+        function save_btn_callback_new()
+        {
+            collection_update_ajax_submit_new('collector-new');
+        }
+
+        function clear_deleted_resources_table(collection_id)
+        {
+            // hide view deleted
+            $("#view_deleted_res_div").hide();
+            $("#clear-deleted-res-btn").hide();
+            $("#clear-deleted-res-warning").show();
+
+            url = "/hsapi/_internal/" + collection_id +"/update-collection-for-deleted-resources/";
+            $.ajax({
+            type: "POST",
+            url: url,
+            dataType: "json",
+            data: {},
+            success: function(result)
+            {
+                /* The div contains now the updated form */
+                json_response = result;
+                if (json_response.status === 'success')
+                {
+                    $('#deleted-resources-table').DataTable().rows().remove().draw(false);
+                    $("#clear-deleted-res-warning").hide();
+                }
+            },
+            error: function(XMLHttpRequest, textStatus, errorThrown)
+            {
+            }
+            });
```

Comment:
move { to its previous line. Also, should users be notified if there is an error when clearing deleted resource from collection? At least you should log the error to the console when there is an error.